src/Kovan-DssSpell.sol

```diff
@@ -67,14 +67,26 @@ contract DssSpellAction is DssAction {
     uint256 constant RAY = 10**27;
     uint256 constant RAD = 10**45;
 
```

Comment:
`VowAbstract` interface can be removed (L24) as not used in the spell.

Reply:
thanks

Reply:
Done